### ISSUE TITLE
Feedback for Install and Set Up SQL

### ISSUE THREAD

**Issue by Paul-Morris** [DNNCommunity/DNNDocs#323]
How exactly do I install SQL. It's not explained in the docs. Do I need to buy it? I seem to already have SQL 2009 running but DNN doesn't list it as compatible

---
#### Document Details

⚠ *Do not edit this section. It is required for dnndocs.com ➟ Core Team processing.*

* Content Source: [set-up-sql](https://github.com/DNNCommunity/DNNDocs/blob/master/content/administrators/setup/set-up-sql/index.md/#L1)

**Comment by valadas** [comment]
Hi @Paul-Morris , I know someone is working on improving that part of the documentation.

In the meantime, here are a couple hints:

SQL 2019 is a preview, I would not recommend using it in production yet.
You can get SQL Server 2017 here https://www.microsoft.com/en-ca/sql-server/sql-server-downloads

Depending on your needs, you have a few options on edition.
- Dowload free trial, this is a trial of the paid version of SQL Server, go ahead if you know you will need the paid version, if not, avoid that button and scroll a bit down :)
- Express edition is free and can be used in production, it has a limit on the number of cpu cores and ram and some such and does not have some advanced tools and features. But it is more than enough for a small to medium sized Dnn instance (if memory serves good, I thinkg it is limited to 10GB per database and a single socket no mater how many cores. Don't quote me on the specs, but more than enough unless you have some huge install.
- Developer edition is the full edition just like the paid one, you get it free for development but it is not licensed for production.

TLDR; Get Express unless you know you need something else.

If you want a video of setting it up, here goes https://www.youtube.com/watch?v=zLB6uzdqGF8&list=PLIx1M8IdVvqZ0bnODGqJyxvONNPj5BzMP&index=1

This is in a playlist, video one sets up the tools (if you are not a developer, you can skip visual studio parts), video 2 sets up Dnn, then it goes on into more development focused stuff. Keep in mind those videos where recorded with development in mind, whatever does not make sense for production, use best judgement.

**Comment by MaiklT** [comment]
Hi @Paul-Morris, I am working on this, but time is a tight resource, so it will last a while. In the meantime, this could help: [facing problem in dnn installation on my local machine
](https://stackoverflow.com/questions/57088302/facing-problem-in-dnn-installation-on-my-local-machine/57096960#57096960). It does not explain how to install SQL Server in detail, but you don't have to care too much about the options when you set it up on your local machine for testing or development purposes.

**Comment by Paul-Morris** [comment]
Thanks guys, I figured it out eventually by using https://www.nvquicksite.com/

I didn't have enough time to setup SQL Server Management Studio and IIS etc but I'll definitely educate myself in the future!
